src/forms/TextInput.jsx

```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import cx from 'classnames';
+import { injectIntl, intlShape } from 'react-intl';
```

Comment:
From slack :) We don't need to inject intl here as this should be handled by the user of the component. It is wrapped in an IntlProvider and provides any translations down to these MWC components so they can stay simple aka dumb components :)